utils/crypto.cpp

```diff
@@ -178,6 +183,111 @@ namespace
             return false;
         }
     }
+
+
+    size_t
+    write_function(void* contents,size_t size, size_t nmemb, void* userp)
+    {
+        reinterpret_cast<std::string*>(userp)->append(reinterpret_cast<char*>(contents), size * nmemb);
+        return size * nmemb;
+    }
+
+
+    bzn::json_message
+    get_curl_response(const std::string& function_call, const std::string& url = ROPSTEN_URL)
+    {
+        const std::string MSG_ERROR_CURL{"curl_easy_perform() failed: "};
```

Comment:
I would put this outside so it's initialized only once.

Reply:
OK.  Done.